abs.yaml

```diff
@@ -3,4 +3,15 @@
 # which conda thinks aren't required
 build_parameters:
   - "--suppress-variables"
-  - "--skip-existing"
+  - "--error-overlinking"
+#  - "--skip-existing"
+
+# macOS 12.3 or above is required for running the GPU variant (MPS support). No way to specify this for only the GPU
+# variant, so it's specified for both.
+extra_labels_for_os:
+  osx-arm64: [ventura]
```

Comment:
does it mean we only build on ventura ?

Reply:
Yes this is what it means.

Reply:
yes, but since the SDK used for CPU is older, it will work on earlier macs